templates/includes/sidebar.html

```diff
@@ -0,0 +1,6 @@
+<ul class="sidebar">
+  <li><a href="{% url 'taxi:index' %}">Home page</a></li>
+  <li><a href="#">Manufactures</a></li>
+  <li><a href="#">Cars</a></li>
+  <li><a href="#">Drivers</a></li>

```

Comment:
The links for 'Manufactures', 'Cars', and 'Drivers' are currently placeholders. Consider updating these with actual URLs or URL template tags once the corresponding views and URL patterns are available.